src/components/Icons/Cart.js

```diff
@@ -16,7 +16,7 @@ const CartIcon = ({ children, color, ...props }) => (
         <rect x="0" y="0" width="24" height="24" />
         <path
           fill={color}
-          {...color !== "currentColor" && { fillOpacity: ".65" }}
+          {...color !== "currentColor" && { fillOpacity: "1" }}
```

Comment:
Hi @BlakeCrowther. For your product's use case, I'd recommend extending this `styled-component` to apply a custom `fill-opacity` style for this component, rather than us making a breaking design change to the component that would impact other products.

Reply:
Do you mean something like this within the next-renderer repo, rather than changing the icon in aurora? 
<img width="351" alt="Screen Shot 2019-08-08 at 11 54 15 AM" src="https://user-images.githubusercontent.com/26388911/62729680-4e099680-b9d3-11e9-982b-404591e8eba3.png">

Reply:
Correct. If this change isn’t required at the global level, then that is the recommended override approach.